routes/mapstats.js

```diff
@@ -66,20 +124,48 @@ router.get("/:match_id", async (req, res, next) => {
   }
 });
 
-/** POST - Create a map stat object from a given match.
- * @name router.post('/create')
- * @memberof module:routes/mapstats
- * @function
- * @param {int} req.body[0].match_id - The ID of the match.
- * @param {int} req.body[0].map_number - The current map number the series is on.
- * @param {string} req.body[0].map_name - The current map name.
- * @param {DateTime} req.body[0].start_time - The start time, as DateTime.
+
+/**
+ * @swagger
  *
+ * /mapstats:
+ *   post:
+ *     description: Add map stats for a match
+ *     produces:
+ *       - application/json
+ *     requestBody:
+ *      required: true
+ *      content:
+ *        application/json:
+ *          schema:
+ *            type: object
+ *            properties:
+ *              match_id:
+ *                type: string
+ *     tags:
+ *       - mapstats
+ *     responses:
+ *       200:
+ *         description: Match stats
+ *         content:
+ *           application/json:
+ *             schema:
+ *               $ref: '#/components/schemas/SimpleResponse'
+ *       400:
+ *         $ref: '#/components/responses/BadRequest'
+ *       403:
+ *         $ref: '#/components/responses/Unauthorized'
+ *       404:
+ *         $ref: '#/components/responses/MatchNotFound'
+ *       422:
+ *         $ref: '#/components/responses/MatchAlreadyFinished'
+ *       500:
+ *         $ref: '#/components/responses/Error'
  */
-router.post("/create", Utils.ensureAuthenticated, async (req, res, next) => {
+router.post("/", Utils.ensureAuthenticated, async (req, res, next) => {
   try {
     if (req.body[0].match_id == null) {
-      res.status(404).json({ message: "Match ID Not Provided" });
+      res.status(400).json({ message: "Match ID Not Provided" });
```

Comment:
Make sure to change the tests as well if you're changing out some of the response codes, the tests under `__test__` do look for some specific status responses to ensure we hit edge cases.